### PR TITLE
DER-83 - Simplify the contract party tree in derivatives

### PR DESCRIPTION
**Pull request by ElisaKendall**
Signed-off-by: Elisa Kendall <ekendall@thematix.com>

## Description

This resolution takes advantage of recent streamlining in FND with respect to contract parties, and takes it a step further to clean up the derivatives contract party hierarchy as well as to simplify out a number of redundant elements (classes and properties) related to Options in particular.

Fixes: #1214 / DER-83

## Checklist:

- [x] I'm familiar with the [FIBO developer quide](../CONTRIBUTING.md#contributing-to-the-fibo-code). My contribution meets all the requirements described there.
- [x] My contribution follows the [principles of best practices for FIBO](../ONTOLOGY_GUIDE.md).
- [x] My changes have been reconciled with latest master and no merge conflicts remain.
- [x] This PR is related to exactly one issue. The issue is referenced by using a GitHub keyword such as "fixes", "closes", or "resolves".
- [x] Hygiene tests have been applied by a PR with "(WIP)" in title.
- [x] The issue has been tested locally using a reasoner (for ontology changes).

### CODE REVIEW

**Review by dsnewman — APPROVED** [review]

File: DER/DerivativesContracts/Options.rdf [hunk 1]
```diff
@@ -60,28 +66,29 @@
 	xmlns:xsd="http://www.w3.org/2001/XMLSchema#">
 	
 	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/Options/">
-		<rdfs:label xml:lang="en">Options</rdfs:label>
+		<rdfs:label xml:lang="en">Options Ontology</rdfs:label>
```
Comment (rivettp) [inline DER/DerivativesContracts/Options.rdf:69]: Do we really need to include "Ontology"? We don't include "Class" in the name of every class!

File: DER/DerivativesContracts/Options.rdf [hunk 2]
```diff
@@ -229,14 +236,7 @@
...
 		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-opt;hasOptionTypeElectionDate"/>
-				<owl:someValuesFrom rdf:resource="&fibo-fnd-dt-fd;Date"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
-		<rdfs:label xml:lang="en">chooser option contract</rdfs:label>
-		<owl:disjointWith rdf:resource="&fibo-der-der-opt;PutOptionContract"/>
+		<rdfs:label xml:lang="en">chooser option</rdfs:label>
```
Comment (rivettp) [inline DER/DerivativesContracts/Options.rdf:239]: no definition?

File: DER/SecurityBasedDerivatives/EquityOptions.rdf [hunk 3]
```diff
@@ -189,14 +189,14 @@
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-opt;definesOptional"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-sbd-eqo;EquityOTCOptionTransaction"/>
+				<owl:onProperty rdf:resource="&fibo-der-sbd-eqo;hasRelevantJurisdiction"/>
```
Comment (rivettp) [inline DER/SecurityBasedDerivatives/EquityOptions.rdf:192]: not a fan of this name - "relevant" for what purpose?

**Review by rivettp — APPROVED** [review]
Great cleanup.

**Pull request merged**